src/JasperBus/IServiceBus.cs

```diff
@@ -21,15 +21,15 @@ public interface IServiceBus
         /// <returns></returns>
         Task<TResponse> Request<TResponse>(object request, RequestOptions options = null);
 
-        void Send<T>(T message);
+        Task Send<T>(T message);
```

Comment:
The main rule that I've lived by with async/await is "top to bottom or nothing".  If we have a sync method making async calls it will inevitably end up with weird problems.  For example, the in-memory queue could potentially apply backpressure if the number of items in the buffer exceeds the limit, in which case the caller needs to be awaiting the Task from Send.